.github/policies/resourceManagement.yml

```diff
@@ -24,6 +24,24 @@ configuration:
       - closeIssue
       - addReply:
           reply: "Closing this issue as we've seen no reply to the request for more information. If you are able to get the requested information, please add it to the issue and we will retriage it. "
+   
+    - description: Close stale PRs
+      frequencies:
+      - daily:
+          hour: 7
+      - daily:
+          hour: 19
+      filters:
+      - isOpen
+      - isPullRequest
+      - not:
+          isDraft
+      - noActivitySince:
+          days: 14

```

Comment:
I'd propose a bit longer than 14 days, since that's probably going to auto-close all our PRs that get opened around Christmas/New Years. :smile:

Reply:
Is it possible to give them a nudge after 2 weeks, then a bit more time to respond before closing?